payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/AfterpayClearpayElementUI.kt

```diff
@@ -31,6 +35,48 @@ import com.stripe.android.ui.core.shouldUseDarkDynamicColor
 fun AfterpayClearpayElementUI(
     enabled: Boolean,
     element: AfterpayClearpayHeaderElement
+) {
+    AfterpayClearpayElementUINew(enabled = enabled, element = element)
+    AfterpayClearpayElementUIOriginal(enabled = enabled, element = element)
+}
+
+@Composable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+fun AfterpayClearpayElementUINew(
+    enabled: Boolean,
+    element: AfterpayClearpayHeaderElement
+) {
+    val context = LocalContext.current
+    val messageFormatString = element.getLabel(context.resources)
+        .replace("<img/>", "<img src=\"afterpay\"/>")
+
+    Html(
+        html = messageFormatString,
+        enabled = false,
+        imageGetter = mapOf(
+            "afterpay" to EmbeddableImage(
+                R.drawable.stripe_ic_afterpay_clearpay_logo,
+                R.string.stripe_paymentsheet_payment_method_afterpay_clearpay,
+                colorFilter = if (MaterialTheme.colors.surface.shouldUseDarkDynamicColor()) {
+                    null
+                } else {
+                    ColorFilter.tint(Color.White)
+                }
+            )
+        ),
+        modifier = Modifier.padding(4.dp, 8.dp, 4.dp, 4.dp),
+        color = MaterialTheme.paymentsColors.subtitle,
+        style = MaterialTheme.typography.h6,
+        urlSpanStyle = SpanStyle(),
+        imageAlign = PlaceholderVerticalAlign.Bottom
+    )
+}
+
+@Composable
```

Comment:
This is just present at the moment to compare before and after and take some screenshots.